hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNode.java

```diff
@@ -1012,6 +1018,10 @@ private void startCommonServices(Configuration conf) throws IOException {
       LOG.info(getRole() + " service RPC up at: "
           + rpcServer.getServiceRpcAddress());
     }
+    if (rpcServer.getLifelineRpcAddress() != null) {
```

Comment:
Is this change related with the issue raised in the jira? It does not seem so.

Reply:
yes , this change is not related to the issue raised in the jira, I will update this when submitting a new jira.